cli/internal/cmd/upgradeapply.go

```diff
@@ -269,6 +274,23 @@ func (u *upgradeApplyCmd) parseUpgradeVars(cmd *cobra.Command, conf *config.Conf
 	}
 }
 
+// handleInvalidK8sPatchVersion checks if the Kubernetes patch version is supported and asks for confirmation if not.
+func handleInvalidK8sPatchVersion(cmd *cobra.Command, version string, yes bool) error {
+	valid := versions.NewValidK8sVersion(version, true) == ""
+
+	if !valid && !yes {
+		confirmed, err := askToConfirm(cmd, fmt.Sprintf("WARNING: The Kubernetes patch version %s is not supported. Do you want to continue anyway?", version))
```

Comment:
```suggestion
		confirmed, err := askToConfirm(cmd, fmt.Sprintf("WARNING: The Kubernetes patch version %s is not supported. If you continue, Kubernetes upgrades will be skipped. Do you want to continue anyway?", version))
```

I would try to explain with a little more detail what saying "yes" actually entails. Feel free to phrase this in a different way!